### PR TITLE
Update rooms-deploy.md

### PR DESCRIPTION
**Pull request by rededwin**
Add links to Teams Rooms advanced deployment guide available on Setup.Microsoft.Com and Admin Center

### CODE REVIEW

File: Teams/rooms/rooms-deploy.md [hunk 1]
```diff
@@ -30,6 +30,9 @@ Deployment of Microsoft Teams Rooms essentially breaks down into phases:
 - (Optional) Setting up Azure Monitor for your systems (see [Deploy Microsoft Teams Rooms management with Azure Monitor](azure-monitor-deploy.md)
 - Setting up Teams Rooms in meeting spaces and connecting the peripheral devices you need (see the OEM documentation for your set of devices)
 
+> [!TIP]
+> As a companion to this article, we recommend using the [**Microsoft Teams Rooms automated setup guide**](https://go.microsoft.com/fwlink/?linkid=2224463) when signed in to the Microsoft 365 Admin Center. This guide will customize your experience based on your environment.  To review best practices without signing in and activating automated setup features, go to the [**M365 Setup portal**](https://go.microsoft.com/fwlink/?linkid=2222974).
```
Comment (JohanFreelancer9) [inline Teams/rooms/rooms-deploy.md:34]:
```suggestion
> As a companion to this article, we recommend using the [Microsoft Teams Rooms automated setup guide](https://go.microsoft.com/fwlink/?linkid=2224463) when signed in to the Microsoft 365 admin center. This guide will customize your experience based on your environment.  To review best practices without signing in and activating automated setup features, go to the [Microsoft 365 setup portal](https://go.microsoft.com/fwlink/?linkid=2222974).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @rededwin, kindly commit the suggested change. Many thanks.

**Comment by rededwin** [comment]
@dstrome PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by scanum** [comment]
@dstrome
PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by rededwin** [comment]
@dstrome when will this be published to public?

**Pull request merged**